src/components/operations/operation-details/ko/runtime/authorization.ts

```diff
@@ -410,15 +414,20 @@ export class Authorization {
         keyParameter.required = true;
         keyParameter.inputType("password");
 
-        this.consoleOperation().request.queryParameters.push(keyParameter);
-        this.updateRequestSummary();
+        if (this.isGraphQL()) {
+            this.queryParameters.push(keyParameter);
```

Comment:
`this.queryParameters.push(keyParameter);` and `this.consoleOperation().request.queryParameters.push(keyParameter);` - aren't they same?

Reply:
Unfortunately no, consoleOperation is a class that we have specifically to handle REST, WebSocket, etc APIs. In this class are included the headers and parameters of the requests of these APIs. In our graphql solution we don't use this class, so we have to handle headers and parameters independently. 

For this reason, the graphql console passes these parameters independently 
![image](https://user-images.githubusercontent.com/88548172/166600295-5e8aac25-a477-4175-a311-b5efed0dfbe5.png)

and the operation console passes the instance of consoleOperation.
![image](https://user-images.githubusercontent.com/88548172/166601554-548d0d13-3f5c-4c06-93a8-6d4363063d41.png)